### PR TITLE
feat: monochrome tray icon

### PR DESCRIPTION
**Pull request by ShGKme**
One of the PRs, continuing https://github.com/nextcloud/talk-desktop/pull/835

### ☑️ Resolves

* https://github.com/nextcloud/talk-desktop/issues/364
* As a part of a new setting, fixes two issues:
  * https://github.com/nextcloud/talk-desktop/issues/446
  * Handle system theme change to change the tray icon theme

### 🖼️ Screenshots

image

Handle system theme change without reload

tray-icon-change

### CODE REVIEW

File: src/talk/renderer/Settings/DesktopSettingsSection.vue [hunk 1]
```diff
@@ -58,6 +59,10 @@ function relaunch() {
 				{{ t('talk_desktop', 'Theme') }}
 			</SettingsSelect>
 
+			<NcCheckboxRadioSwitch :checked.sync="monochromeTrayIcon" type="switch">
```
Comment (Antreesy) [inline src/talk/renderer/Settings/DesktopSettingsSection.vue:62]: Requires a restart for me to take effect

**Review by Antreesy — APPROVED** [review]
Works, tested on Win

File: src/talk/renderer/Settings/DesktopSettingsSection.vue (hunk 1 above)
Comment (ShGKme) [reply src/talk/renderer/Settings/DesktopSettingsSection.vue:62]: Oops, i forgot

Comment (Antreesy) [reply src/talk/renderer/Settings/DesktopSettingsSection.vue:62]: Should be good now, you can merge)

**Pull request merged**